config/initializers/omniauth.rb

```diff
@@ -1,3 +1,6 @@
+require 'openssl'
+OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
```

Comment:
Can we move this into `config/environments/development.rb`? We definitely shouldn't be skipping certificate verification in production.